hurumap/settings.py

```diff
@@ -302,6 +302,7 @@
                 "#8ecc23", "#fccd06", "#dbba97", "#aaaaaa"
             ],
         },
+        'show_y_axis': False,
```

Comment:
@KhadijaMahanga if this is going to be used in JS, doesn't it need to be `false`; all lowercase 🤔

Reply:
In Django templates, we use `yesno` filter to convert it into JS compatible `true` or `false` @phillipahereza : https://github.com/CodeForAfrica/HURUmap/blob/f288c88ac0a2f3874943ae236d3be513f143aaed/hurumap/templates/profile/profile_detail.html#L989

And I'm assuming when we're converting it to JS in `settings_js.html` we use `jsonify` filter which should handle the conversion. @KhadijaMahanga ?